### PR TITLE
Decrypt environment variables with kms

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: test/test_matcher.py [hunk 1]
```diff
@@ -16,7 +17,14 @@ def aws_credentials():
...
     os.environ['AWS_SESSION_TOKEN'] = 'testing'
 
+
 @pytest.fixture(scope='function')
+def kms(aws_credentials):
+    with mock_kms():
+        yield boto3.client('kms', region_name='eu-west-2')
+
+
+@pytest.fixture(scope='function')  #
```
Comment (suzannehamilton) [inline test/test_matcher.py]: Is the hash at the end of the line a typo?

File: test/test_matcher.py [hunk 2]
```diff
@@ -98,15 +106,32 @@ def get_records(num=1):
...
+    )['CiphertextBlob']).decode('utf-8')
+
+
+def set_environment(kms):
+    key = kms.create_key(
+        Policy='string',
+        Description='string',
+    )['KeyMetadata']['KeyId']
+    print(encrypt(key, kms, "intg"))
+    os.environ["ENVIRONMENT"] = encrypt(key, kms, "intg")  # intg
```
Comment (suzannehamilton) [inline test/test_matcher.py]: Is the comment necessary here? It looks like it was supposed to explain a base64-encoded value, but it's actually a plain text value?

File: test/test_matcher.py (hunk 1 above)
Comment (MancunianSam) [reply test/test_matcher.py]: Yep, I've taken it out.

File: test/test_matcher.py (hunk 2 above)
Comment (MancunianSam) [reply test/test_matcher.py]: Yeah you're right, I tried it with the base64 encoded values but it didn't work so  I changed it to work like this. I've removed the comment.

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**